internal/server/http/handler/monitoring.go

```diff
@@ -0,0 +1,26 @@
+package handler
```

Comment:
we need to work on this file to

Reply:
I agree, as discussed, let's do that later, thanks :)